supertokens_python/recipe/thirdparty/providers/__init__.py

```diff
@@ -18,4 +18,4 @@
 from .google import Google  # type: ignore
 from .google_workspaces import GoogleWorkspaces  # type: ignore
 from .bitbucket import Bitbucket  # type: ignore
-from .gitlab import GitLab  # type: ignore
+from .gitlab import Gitlab  # type: ignore
```

Comment:
why do we use type: ignore here?

Reply:
I think Satvik used it because these imports aren't being used. 

But actually, it's not the best way. I'll update this.